addons/mail/static/src/owl/components/activity/activity.scss

```
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
+}
+
+.o_Activity_detailsUserAvatar {
+    margin-right: 5px;
```

For **margin-left** or **margin-right**, when it depends on the direction of the text, this would be correct assuming `ltr` but will fail when using `rtl`.

So when the margin indeed depends on the direction of the text (which is 99% of the time), we need to use `margin-inline-start` or `margin-inline-end`.

```suggestion
    margin-inline-end: map-get($spacers, 2);
```

Note that maybe here is a bad example, the question to task is do we want the image to be on the left or on the right of the text when the text direction is inversed?